sdk/appconfiguration/perf-tests/app-configuration/test/index.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { PerfStressProgram, selectPerfStressTest } from "@azure/test-utils-perfstress";
+
+// Expects the .env file at the same level
+import * as dotenv from "dotenv";
+import { ListSettingsTest } from "./listSettings.spec";
+dotenv.config();
```

Comment:
Should this just be done in the AppConfigTest class?

Reply:
Moved, doesn't matter either way though.